### DISCUSSION TITLE
2024/03/24/unflowerable_Tulips_and_plant_Vernalization/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#18] [General]
# 2024/03/24/unflowerable_Tulips_and_plant_Vernalization/

寒假那段时间，实验室的一位师姐不知道从哪里弄来了一颗郁金香的球茎，种在了实验室的阳台上。按照郁金香的花期，这段时间它应该已经开出了花——可是并没有，那颗可怜的球茎只是一味的长叶子，长得长长的就跟蒜苗一样高，甚至连个花苞都没打，一点开花的迹象都没有。 在排除了光照（实验室光线很充足）、营养（应该不缺吧？）、水分（可充足了）等影响因素之后，这颗球茎不开花的最大可能原因就是温度了。查询了一些种植经验帖，

https://wz.anoms.top/2024/03/24/unflowerable_Tulips_and_plant_Vernalization/

**Comment by Hazelandinstant** [comment]
哇！如果植物老师当初是小胖是不是就有趣多了！（忽然感叹）

**Reply by cyclinbox** [reply]
呜呜，一点小爱好班门弄斧罢了，完全比不过那些植物学老师（捂脸）
